docs/articles/lavalink/intro.md

```diff
@@ -1,7 +1,85 @@
 # Lavalink - the newer, better way to do music
-Lavalink is a standalone program, written in Java, using JDA. It's a 
+[Lavalink](https://github.com/Frederikam/Lavalink) is a standalone program, written in Java, using JDA. It's a 
 lightweight solution for playing music from sources such as YouTube or 
 Soundcloud. Unlike raw voice solutions, such as VoiceNext, Lavalink can handle 
 hundreds of concurrent streams, and supports sharding.
 
-TODO
\ No newline at end of file
+## Configuring Java
+In order to run Lavalink, you must have Java 11 or greater installed.
+The latest releases can be found [here](https://www.oracle.com/technetwork/java/javase/downloads/index.html).
+
+Make sure the location of the newest JRE's bin folder is added to your system variable's path. This will make the `java` command run from the latest runtime. You can verify that you have the right version by entering `java -version` in your command prompt or terminal.
+
+## Downloading Lavalink  
+Next, head over to the [releases](https://ci.fredboat.com/viewLog.html?buildId=lastSuccessful&buildTypeId=Lavalink_Build&tab=artifacts&guest=1) tab on the Lavalink Github page. and download the Jar file from the latest version.
```

Comment:
Or their [GitHub releas page](https://github.com/Frederikam/Lavalink/releases).

Reply:
Yeah I originally had that, but the latest build there has a bug with lavaplayer that prevents users from playing any tracks. They resolved it in the latest FredBoat CI build.

Reply:
I'm aware - I use the CI builds myself. I just think it's worth mentioning both and giving the reasoning.